### ISSUE TITLE
Add aria-labels + titles (+ popups?) to control buttons

### ISSUE THREAD

**Issue by marcustyphoon** [AprilSylph/XKit-Rewritten#973]
### Concept

Right now, if one forgets what one of XKit's control buttons is, there's no hovertext to indicate this. This should be easy to fix. Also, accessibility, of course.

Edit: I just had a vision of popping up a notification when you register a control button without one of these accessibility attributes being set, but only in developer mode. We've already gone down that road though :D

**Comment by marcustyphoon** [comment]
Just dawned on me that we could also pretty easily replicate the control button tooltips (and then disable our in the tweak that hides the Tumblr ones, naturally).

https://nicolasgallagher.com/pure-css-speech-bubbles/demo/
